postBuild

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
I think it's worth calling out that this file is to support Binder.  A comment with a link to https://mybinder.readthedocs.io/en/latest/using/config_files.html#postbuild-run-code-after-installing-the-environment is probably the simplest way to do that.

Reply:
Good idea! Done